hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml

```diff
@@ -14,7 +14,8 @@ market: null
 # What leverage to used
 leverage: null
 
-# Position mode to use, one-way or hedge position mode
+# Position mode to use, hedge mode or one-way position mode (true/false).
+# If value is set to true, hedge mode is enabled.
```

Comment:
```suggestion
# Position mode to use, hedge mode or one-way position mode.
```
I addressed this issue in the PR to add position management. So users now have to specify either "One-way" or "Hedge" for position mode rather that True/False.